fang/Cargo.toml

```diff
@@ -61,19 +73,9 @@ optional = true
 version = "0.15"
 optional = true
 
-[dependencies.bb8-postgres]
-version = "0.8"
-features = ["with-serde_json-1" , "with-uuid-1" , "with-chrono-0_4"]
-optional = true
-
-[dependencies.postgres-types]
-version = "0.X.X"
-features = ["derive"]
-optional = true
-
 [dependencies.tokio]
 version = "1.25"
-features = ["rt", "time", "macros"]
+features = ["rt", "time", "macros"]#, "tracing"]
```

Comment:
this change is not needed

Reply:
I added `tracing` feature because it is awesome to debug `tokio` based programs. 

Could we leave this commented and dis-comment it when it is needed ?